src/stream/src/executor/lookup/cache.rs

```diff
@@ -45,12 +44,17 @@ impl LookupCache {
             let key = row.project(arrange_join_keys).into_owned_row();
             if let Some(mut values) = self.data.get_mut(&key) {
                 // the item is in cache, update it
+                let row = row.into_owned_row();
                 match op {
                     Op::Insert | Op::UpdateInsert => {
-                        values.insert(row.into_owned_row());
+                        if !values.insert(row) {
+                            panic!("inserting a duplicated value");
```

Comment:
It appears that we can now safely replace `panic` with `inconsistency_panic` after the refactoring.

Reply:
Yes, exactly.